### PR TITLE
fix: cli usage info

### PR DESCRIPTION
**Pull request by sqin2019**

### CODE REVIEW

**Review by sailorlqh — APPROVED** [review]

**Review by sethvargo — APPROVED** [review]

File: pkg/cli/cli_handle.go [hunk 1]
```diff
@@ -50,23 +50,21 @@ func (c *CLIHandleCommand) Help() string {
 	return `
 Usage: {{ COMMAND }} [options]
 
-Run "do" commands in the CLI request YAML file at the given path:
+Handle "do" commands in the CLI request YAML file at the given path:
 
-      aod cli do -path "/path/to/file.yaml"
+      aod cli handle -path "/path/to/file.yaml"
```
Comment (sethvargo) [inline pkg/cli/cli_handle.go:55]: You can also do `{{ COMMAND }}`

Comment (sqin2019) [reply pkg/cli/cli_handle.go]: Updated.

File: pkg/cli/cli_handle.go [hunk 2]
```diff
@@ -52,19 +52,19 @@ Usage: {{ COMMAND }} [options]
 
 Handle "do" commands in the CLI request YAML file at the given path:
 
-      aod cli handle -path "/path/to/file.yaml"
+      aod cli {{ COMMAND }} -path "/path/to/file.yaml"
```
Comment (sethvargo) [inline pkg/cli/cli_handle.go:55]: `COMMAND` is the _full_ command

```suggestion
      {{ COMMAND }} -path "/path/to/file.yaml"
```

**Comment by capri-xiyue** [comment]
Such CLI looks like the after the `do` process, it cleans up some left resources, similar examples https://github.com/helm/helm/issues/1920 and https://github.com/transferwise/pipelinewise/issues/573

**Review by capri-xiyue — APPROVED** [review]

**Review by capri-xiyue — APPROVED** [review]

**Pull request merged**